src/main/java/winstone/Launcher.java

```diff
@@ -442,15 +449,23 @@ public static void initLogger(Map<String, String> args) throws IOException {
         // boolean showThrowingLineNo = Option.LOG_THROWING_LINE_NO.get(args);
         boolean showThrowingThread = Option.LOG_THROWING_THREAD.get(args);
         OutputStream logStream;
+        Charset logCharset;
         if (args.get("logfile") != null) {
-            logStream = new FileOutputStream(args.get("logfile"));
+            try {
+                logStream = Files.newOutputStream(Paths.get(args.get("logfile")), StandardOpenOption.CREATE, StandardOpenOption.APPEND);
+            } catch (InvalidPathException e) {
+                throw new IOException(e);
+            }
+            logCharset = StandardCharsets.UTF_8;
```

Comment:
Technically a change in behavior, but I think it makes the most sense for this log to be in UTF-8.

Reply:
as it's a breaking change the PR should have the appropriate label...

Reply:
> as it's a breaking change the PR should have the appropriate label...

I updated the label and the changelog to mention this change.